types/context.d.ts

```diff
@@ -1,7 +1,7 @@
 import { ContextConfigDefault } from './utils'
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface FastifyContextConfig {
+  url: string;
```

Comment:
this seems wrong. 

The corresponding code line would be imho:
https://github.com/fastify/fastify/blob/d93adec5b7403a52a254ddd7d2eeeef59ac857fa/lib/route.js#L274

So it should be extending Partial<RouteOptions> or something like this